ArmPlatformPkg/PeilessSec/PeilessSec.inf

```diff
@@ -51,6 +51,7 @@
   PrintLib
   SerialPortLib
   TimerLib
+  StackCheckLib

```

Comment:
This PR adds StackCheckLib to all modules of type SEC.  This is not listed explicitly in the PR Description Integration Instructions.  Is this really required, or do the instructions need to be updated?

Reply:
This is really needed. With the new approach (i.e. not linking StackCheckLib as a NULL lib) we rely on the entry point libs being linked to every module to provide the StackCheckLib definitions. SEC does not have a standard entry point lib, each SEC module implements their own entry point. As such, with this approach, we need every SEC module to link against StackCheckLib to get the compiler inserted externs resolved. I will update the integration instructions.